tools/editor/project_manager.cpp

```diff
@@ -1103,23 +1103,23 @@ void ProjectManager::_install_project(const String& p_zip_path,const String& p_t
 }
 
 void ProjectManager::_files_dropped(StringArray p_files, int p_screen) {
-	Set<String> folders;
+	Set<String> folders_set;
 	DirAccess *da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
 	for (int i = 0; i < p_files.size(); i++) {
 		String file = p_files[i];
-		folders.insert(da->dir_exists(file) ? file : file.get_base_dir());
+		folders_set.insert(da->dir_exists(file) ? file : file.get_base_dir());
 	}
 	memdelete(da);
-	multiple_folders.resize(0);
-	if (folders.size()>0) {
-		for (Set<String>::Element *E=folders.front();E;E=E->next()) {
-			multiple_folders.append(E->get());
+	if (folders_set.size()>0) {
+		StringArray folders;
```

Comment:
If you drag 10 files from one folder you get 10 equals folder paths in list(array,vector), and only one path in set.